stripe/request_metrics.py

```diff
@@ -0,0 +1,10 @@
+class RequestMetrics(object):
```

Comment:
Minor note, but the one in Ruby is called `StripeRequestMetrics`, and it'd be nice to standardize on a name. `RequestMetrics` is probably better given it's already in the Stripe namespace.

Reply:
I'm fine either way, I just figured `from stripe.stripe_request_metrics import StripeRequestMetrics` was overly verbose.

Reply:
Yeah, the `Stripe` prefix seems pretty redundant. Let's stick with what you have.